src/cbc_sdk/endpoint_standard/base.py

```diff
@@ -417,6 +418,46 @@ def _get_query_parameters(self):
 
         return args
 
+    def set_rows(self, rows):
+        """
+        Sets the 'rows' query body parameter, determining how many rows of results to request.
+        Args:
+            rows (int): How many rows to request.
+        """
+        if not isinstance(rows, int):
+            raise ApiError(f"Rows must be an integer. {rows} is a {type(rows)}.")
+        self._batch_size = rows
+        self._default_args["rows"] = self._batch_size
+        return self
+
+    def set_time_range(self, start=None, end=None, window=None):
+        """
+        Sets the 'time_range' query body parameter, determining a time window based on 'device_timestamp'.
+        Args:
+            start (str in ISO 8601 timestamp): When to start the result search.
+            end (str in ISO 8601 timestamp): When to end the result search.
+            window (str): Time window to execute the result search, ending on the current time.
+                Should be in the form "-2w", where y=year, w=week, d=day, h=hour, m=minute, s=second.
+        Note:
+            - `window` will take precendent over `start` and `end` if provided.
+        Examples:
+            query = api.select(Event).set_time_range(start="2020-10-20T20:34:07Z")
+            second_query = api.select(Event).set_time_range(start="2020-10-20T20:34:07Z", end="2020-10-30T20:34:07Z")
+            third_query = api.select(Event).set_time_range(window='-3d')
```

Comment:
```suggestion
            query = api.select(EnrichedEvent).set_time_range(start="2020-10-20T20:34:07Z")
            second_query = api.select(EnrichedEvent).set_time_range(start="2020-10-20T20:34:07Z", end="2020-10-30T20:34:07Z")
            third_query = api.select(EnrichedEvent).set_time_range(window='-3d')
```
Adapt after copy/paste ;)

Reply:
Uh oh...fixed :)

Reply:
Eheh ;) You can close this too!